server/src/main/resources/rules/rules.js

```diff
@@ -2184,6 +2187,9 @@ var Autobind = {
                         }
                     }
                     this.pools = pools_without_bad_attrs;
+                    log.debug("validate - Removed {} of {} pools from stack '{}'",
+                    initial_pool_length - this.pools.length, initial_pool_length, this.stack_id);
```

Comment:
When you get to three fields, this needs to be an array for backwards compatibility.

Reply:
Thanks, will fix this now. Surprisingly I've tested autoattatch with these rules in candlepin-2.1 and there was no problem. Which version is the correct to test against for this compatibility?

Reply:
fixed